src/app/Dashboard/AutomatedAnalysis/Filters/AutomatedAnalysisNameFilter.tsx

```diff
@@ -36,33 +36,36 @@
  * SOFTWARE.
  */
 
-import React from 'react';
-import { Select, SelectOption, SelectVariant } from '@patternfly/react-core';
 import { CategorizedRuleEvaluations } from '@app/Shared/Services/Report.service';
+import { Select, SelectOption, SelectVariant } from '@patternfly/react-core';
+import React from 'react';
 
 export interface AutomatedAnalysisNameFilterProps {
   evaluations: CategorizedRuleEvaluations[];
   filteredNames: string[];
   onSubmit: (inputName: string) => void;
 }
 
-export const AutomatedAnalysisNameFilter: React.FunctionComponent<AutomatedAnalysisNameFilterProps> = (props) => {
+export const AutomatedAnalysisNameFilter: React.FunctionComponent<AutomatedAnalysisNameFilterProps> = ({
+  onSubmit,
+  ...props
```

Comment:
Actually, on this note, I notice we are destructing callbacks but leave others in props. Is this true and the recommended way?

Reply:
Oh, yes. There is no mention of a recommended way other than to destructure ALL the props, but eslint doesn't complain this way. See the other comment for more information.